content/en/about-filecoin/managing-assets/index.md

```diff
@@ -9,13 +9,31 @@ weight: 60
+
+With non-custodial wallets, the user holds their own private key and has total control over the movement of assets. Non-custodial wallet users are also solely responsible for securing their assets. Protecting private keys and seed phrases is extremely important to prevent fraudulent access to assets. Most non-custodial wallets do not have KYC requirements and allow users to maintain anonymity.
+
+### Hot vs cold
+
+Hot wallet refers to any wallet which is connected to a network. They can be mobile, desktop, or browser based. Connecting to a network comes with its own security concerns, therefore it is recommended to keep large balances in cold wallets and use hot wallets to hold funds you need to access frequently.
+
+Cold wallet most commonly refers to hardware wallet devices. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key which makes them more resistant to loss via theft. Transaction signing happens within the device itself and private keys are never exposed to the network. Cold wallets can take longer to access and are most useful for storing larger balances securely. 
+
+### Security
+
+Wallets that have gone through an audit have had their codebase checked by a recognized security firm for security vulnerabilities and potential leaks. However, just because a wallet has had an audit does not mean that it’s 100% bug-proof. Be incredibly cautious when using unaudited wallets.
+
+Never share your seed phrase, password, or private keys. Bad actors will often use social engineering tactics such as phishing emails or posing as customer service or tech support to lure users into handing over their private key or seed phrase.
+
 Wallets allow you to manage your assets without having to meet the [relatively high specifications required to run a Lotus node]({{< relref "set-up" >}}) for Filecoin. Here are some wallets that you might consider.
 
-{{< alert icon="warning" >}}
-**Unaudited Wallets**
+We do not provide support for any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet.
 
-We do not support any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet. Wallets that _have_ gone through an audit have had their codebase checked by a recognized security firm for security vulnerabilities and potential leaks. However, just because a wallet has had an audit does not mean that it's 100% bug-proof. Be incredibly cautious when using unaudited wallets.
-{{< /alert >}}
 
  Name                                           | Description                                                                                                                          | Audited |
```

Comment:
Let's move the wallets list above the tradeoffs (hot vs cold, security, etc.)

Reply:
Also, let's make the text in this list more user-friendly. Can wait til 2nd round of revisions.

Reply:
Also, let's make the text in this list more user-friendly. Can wait til 2nd round of revisions.

Reply:
Moved list up in doc